src/rocketData.py

```diff
@@ -15,127 +13,137 @@ class RocketData:
                 'humidity': float,
                 'pressure': float,
             },
-            'bme': {
-                'yaw': float,
-                'pitch': float,
-                'roll': float,
-                'linear_acceleration': float,
-                'linear_velocity': float
-            },
-            'strain_gauges': {
-                1: float,
-                2: float,
-                3: float,
-                4: float,
-                5: float,
-                6: float,
-                7: float,
-                8: float,
-                9: float,
-                10: float,
-                11: float,
-                12: float
+            'lsm': {
+                'acceleration_x': float,
+                'acceleration_y': float,
+                'acceleration_z': float,
+                'magentometer_x': float,
+                'magentometer_y': float,
+                'magentometer_z': float,
+                'gyroscope_x': float,
+                'gyroscope_y': float,
+                'gyroscope_z': float,
+                'temperature': float
             },
+            'strain_gauges': [
+                float,
+                float,
+                float,
+                float,
+                float,
+                float,
+                float,
+                float,
+                float,
+                float,
+                float,
+                float
+            ],
             'encoders': {
                 'position': float
             },
             'time_stamp': float
         }
 
-    # @property decorator makes the function below a getter 
+    # @property decorator makes the function below a getter
     @property
     def data(self):
         return self._data
-    
-    @property
-    def imu(self):
-        return self._data['imu']
-
-    @imu.setter
-    def imu(self, vals):
-        t, h, p = vals
-        self._data['imu']['temperature'] = t
-        self._data['imu']['humidity'] = h
-        self._data['imu']['pressure'] = p
-
-    @imu.setter
-    def imu_temp(self, t):
-        self._data['imu']['temperature'] = t
 
+    @property
+    def lsm(self):
+        return self._data['lsm']
+
+    @lsm.setter
+    def lsm(self, vals):
+        if len(vals) == 10:
+            ax, ay, az, mx, my, mz, gx, gy, gz, t = vals
+            self._data['lsm']['acceleration_x'] = ax
+            self._data['lsm']['acceleration_y'] = ay
+            self._data['lsm']['acceleration_z'] = az
+            self._data['lsm']['magentometer_x'] = mx
+            self._data['lsm']['magentometer_y'] = my
+            self._data['lsm']['magentometer_z'] = mz
+            self._data['lsm']['gyroscope_x'] = gx
+            self._data['lsm']['gyroscope_y'] = gy
+            self._data['lsm']['gyroscope_z'] = gz
+            self._data['lsm']['temperature'] = t
+        else: 
+            print("LSM accepts 10 values in this order: Acceleration (x, y, z), magentometer (x, y, z), gyroscope (x, y, z")
```

Comment:
```suggestion
            print("LSM accepts 10 values in this order: Acceleration (x, y, z), magentometer (x, y, z), gyroscope (x, y, z)")
```
forgot a closing brace